### ISSUE TITLE
Google Play Store opens to a game every 2 secounds !![solved]!!

### ISSUE THREAD

**Issue by SlySalimander** [dhowe/AdNauseam#981]
### Describe the issue

This ad-on is constantly loading a game in the google play store.  I had to uninstall it.  I was unable to use Firefox in my Android Phone.  Any and all websites I visit.  The Google Play Store will open and it will go to a game.  Even if you back out it will keep forwarding to this game in the Play Store every 2 seconds.

I uninstalled this ad-on tested Firefox and then reinstalled the add-on just to make sure it was the cause.

I tried uninstalling all other extensions.  I had only the AdNauseam add-on installed and no others.  Even in this state the problem continued.

### One or more specific URLs where the issue occurs

Any and all websites are affected by this bug.

### Screenshot in which the issue can be seen

This is the Game that keeps loading in the play store.  Always the same game.  I have never played this game.
http://imgur.com/X5pyscp

### Steps for anyone to reproduce the issue

Install the AdNauseam add-on on an android phone in FireFox.  It will start happening immediately.  Go to any website and it will continue to happen.

Uninstall the add-on and it will stop.

### Your settings

- OS/version:  Android 7.1.2
- Browser/version:  FireFox 52.0.2
- AdNauseam version:  3.0.50
- Phone:  NEXUS 6P
- Other extensions you have installed:  HTTPS Everywhere 5.2.14 ; LastPass 3.3.4 ; Privacy Badger 2017.3.22

**Comment by dhowe** [comment]
Thanks for the report, @SlySalimander 
If you have a moment, can you try the same test with only 'uBlock Origin' installed ?

**Comment by SlySalimander** [comment]
uBlock Origin does not cause the problem.  I tried it just like you said with uBlock Origin only. No problem.  Then I reinstalled the apps I listed originally.  Still no problem. Then I tried installing AdNauseam again.  The problem started immediately and I had to remove it.

**Comment by dhowe** [comment]
Thanks. We are looking into it...

**Comment by cqx931** [comment]
OS/version: Android 6.0.1
Browser/version: FireFox 52.0.2
AdNauseam version: 3.0.50

I have tested this with the settings above but was not able to reproduce the issue.
I have also tried to install other extensions you listed with adnauseam, but the issue still doesn't happen to me.

**Comment by SlySalimander** [comment]
Ok thank you so much for looking so quickly.
I will be able to experiment some more latter on today.
I will try more troubleshooting to narrow this down.

**Comment by SlySalimander** [comment]
Ok I came back here.  I found a solution.

You see after you install the add-on you normally get this.
http://imgur.com/9DCT2Lf
This allows you to set the settings in AdNauseam.
My problem was that this is not what was loading.  Instead I was getting forwarded to the game in the app store.
At least that is what I think was happening.

How did I solve it?

Well I had to do more then just reinstall all of my add-ons.  That did not solve the problem.  After more experimenting today I found this as the solution.

Step 1: Completely remove FireFox from the phone.
Step 2: Run a cleanup utility tool (cclean).
Step 3: Then I reset my phone.
Step 4: Then I reinstalled FireFox.  Then I reinstalled the add-ons that I like to use including AdNauseum.
When I did all of this the problem did not return.

Sorry to suggest this was a AdNauseam problem.  I had this problem for a few days and my initial troubleshooting seemed to point to AdNauseam.

I now think that FireFox was corrupted in some way but for some reason it was only affecting AdNauseam.

I do not know how to recreate the problem anymore.  After doing all of the experimenting today I am not able to recreate the problem anymore.
